dp-cluster-setup-utility.py

```diff
@@ -1497,7 +1500,21 @@ def update_profiler_agent_service_def(self, current_stack_version):
 
     self._chown_to_knox(service_dir)
     self._chown_to_knox(os.path.dirname(service_dir))
+  
+  def update_ambari_service_def(self, current_stack_version):
+    dest_services_base_dir = self._check_dir('/var/lib/knox/data-%s/services' % current_stack_version, 'service')
+    service_dir = '%s/ambari/0.2.2.0' % dest_services_base_dir
+    if os.path.isdir(service_dir):
+      print 'Service files already exist in %s' % service_dir
+    else:
+      os.makedirs(service_dir)
 
+    self._create_service_file(service_dir, 'rewrite.xml')
```

Comment:
_create_service_file is currently hardcoded to look for source files under profiler-agent directory. It should be parameterized. This PR should have failed on testing. Can you please confirm the changes with testing?

Reply:
modified the code to pass service and version and then work accordingly